src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
-body {
-  margin: 0;
-}
+@import './block/@media';
+@import './block/*';
+@import './block/html';
+@import './block/body';
+@import './block//header';
```

Comment:
There is a typo in the import path for the header block. It currently uses a double slash ('//'), which should be corrected to a single slash ('/') to ensure the file is imported correctly.